src/Domain/Creators/CylindricalBinaryCompactObject.cpp

```diff
@@ -663,8 +663,8 @@ Domain<3> CylindricalBinaryCompactObject::create_domain() const {
   if (not time_dependence_->is_none()) {
     for (size_t block = 0; block < number_of_blocks_; ++block) {
       domain.inject_time_dependent_map_for_block(
-          block,
-          std::move(time_dependence_->block_maps(number_of_blocks_)[block]));
+          block, std::move(time_dependence_->block_maps_grid_to_inertial(
+                     number_of_blocks_)[block]));
```

Comment:
Eventually CylindricalBinaryCompactObject should have its own maps, just like the other BinaryCompactObject domain.  It really doesn't make sense for it to use TimeDependence at all, because maps will be different in different Blocks in a way that is specific to this DomainCreator.

So I just made the minimal changes to get it to compile.  If you want, I'm happy to add the distorted frame stuff here, but it won't be used.

Reply:
Nah I don't think it's worth it or necessary here. Thanks for clarifying :)